experiments/jz/joint_training_toy/joint_training_toy1/03_create_dataset.slurm

```diff
@@ -44,14 +44,12 @@ REPO_DIR="${DATASETS_CUSTOM}/${myarray[-1]}"
 
 # Now we launch the script that will perform the preprocessing of the dataset
 # Feel free to add any arguments you like (change me!)
-python bsmetadata/train.py \ # (change me and remove the comment! if you have a specific script)
+python bsmetadata/train.py \
     data_config.experiment="with_metadata" \
-    data_config.metadata_config.metadata_list=["html"] \ # (change me and remove the comment!)
+    data_config.metadata_config.metadata_list=["entity", "timestamp", "url", "website_description"] \
```

Comment:
Have you tested this command locally? I am not sure if this format is valid.

Reply:
I updated it. It seems to work now. Thank you for reviewing.